src/docs/environment.mdx

```diff
@@ -170,6 +172,106 @@ If you would like to be able to run `devserver` outside of your root checkout, y
   displayed correctly.
 </Alert>
 
+## Setting up Getsentry
+
+<Alert title="Corporate Users Only" level="warning">
+  Only Sentry employees have access to getsentry.
+</Alert>
+
+Now that you have sentry all set up, it's time to set up getsentry. Getsentry is
+the application we deploy to production and contains additional functionality
+only offered in our SaaS offering.
+
+Let's start off by cloning the `getsentry` repository to be adjacent with your
+sentry repository:
+
+```bash
+# Go to where you have sentry and clone getsentry.
+cd /path/to/sentry
+cd ..
+git clone git@github.com:getsentry/getsentry.git
+```
+
+It's necessary to keep getsentry in an adjacent directory (it's expected by
+`install-develop-requirements` in the getsentry Makefile). For example, if you
+did a `ls ~/code` you'd see something like:
+
+```bash
+sentry/   getsentry/
+```
+
+Next, create a virtualenv for `getsentry`, and install its dependencies:
+
+```bash
+# Create and source a virtualenv
+
+python -m virtualenv .venv
+source .venv/bin/activate
+
+# Install dependencies
+# Note that you can safely ignore any warnings from pip
+# about "you'll have version ... which is incompatible"
+# Regarding google cloud dependencies.
+
+make develop
+
+# Initialize and bootstrap the database
+make reset-db
+
+# Run the development server
+getsentry devserver --workers
+```
+
+Once you've booted the server you'll access the development environment
+via [http://dev.getsentry.net:8000](http://dev.getsentry.net:8000/), which will
```

Comment:
I don't have enough context on this, but... I feel like this introduces unnecessary friction? I'd expect getsentry vs sentry dev to be pretty much the same, so getsentry devserver should be accessible via `localhost:8000`. cc @dcramer

Reply:
The current getsentry configuration has `PARENT_HOST` set to dev.getsentry.net, and `options[system.url-prefix] =  "dev.getsentry.net:8000"` so I think this is required right now. We could streamline that process in the future though.